schemas/com.snowplowanalytics.travel/booking_complete/jsonschema/1-0-0

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "type": "object",
+  "self": {
+    "vendor": "com.snowplowanalytics.travel",
+    "name": "booking_complete",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Event for when a booking process has been completed",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "ID of the booking"
+    },
+    "type": {
+      "type": "string",
+      "enum": ["experience", "destination"],
+      "description": "Type of booking"
+    },
+    "title": {
+      "type": "string",
+      "description": "Title of the booking"
+    },
+    "location": {
+      "type": "string",
+      "description": "Location (physical) of the booking"
+    },
+    "price": {
+      "type": "number",
+      "description": "Total price of the booking"
+    },
+    "booking_start_date": {
+        "type": "string",
+        "description": "Start date of the booking"
+    },
+    "booking_end_date": {
+        "type": "string",
+        "description": "End date of the booking (optional)"
+    },
+    "booking_start_time": {
+        "type": "string",
+        "description": "Start time of the booking (optional)"
+    },
+    "booking_end_time": {
+        "type": "string",
+        "description": "End time of the booking (optional)"
+    },
```

Comment:
I assume the split is so time can be optional? Should they be locked down with `pattern` to look like the `date-time`-`format` components?